src/main/kotlin/no/nav/arbeidsgiver/min_side/userinfo/UserInfoController.kt

```diff
@@ -0,0 +1,161 @@
+
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.supervisorScope
+import no.nav.arbeidsgiver.min_side.config.GittMiljø
+import no.nav.arbeidsgiver.min_side.controller.AuthenticatedUserHolder
+import no.nav.arbeidsgiver.min_side.models.Organisasjon
+import no.nav.arbeidsgiver.min_side.services.altinn.AltinnService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class UserInfoController(
+    private val altinnService: AltinnService,
+    private val authenticatedUserHolder: AuthenticatedUserHolder,
+    gittMiljø: GittMiljø,
+) {
+
+    val tjenester = mapOf(
+        "ekspertbistand" to Altinnskjema(
+            tjenestekode = "5384",
+            tjenesteversjon = "1",
+        ),
+        "inntektsmelding" to Altinnskjema(
+            tjenestekode = "4936",
+            tjenesteversjon = "1",
+        ),
+        "utsendtArbeidstakerEØS" to Altinnskjema(
+            tjenestekode = "4826",
+            tjenesteversjon = "1",
+        ),
+        "arbeidstrening" to NAVTjeneste(
+            tjenestekode = "5332",
+            tjenesteversjon = gittMiljø.resolve(
+                prod = { "2" },
+                other = { "1" },
+            ),
+        ),
+        "arbeidsforhold" to NAVTjeneste(
+            tjenestekode = "5441",
+            tjenesteversjon = "1",
+        ),
+        "midlertidigLønnstilskudd" to NAVTjeneste(
+            tjenestekode = "5516",
+            tjenesteversjon = "1",
+        ),
+        "varigLønnstilskudd" to NAVTjeneste(
+            tjenestekode = "5516",
+            tjenesteversjon = "2",
+        ),
+        "sommerjobb" to NAVTjeneste(
+            tjenestekode = "5516",
+            tjenesteversjon = "3",
+        ),
+        "mentortilskudd" to NAVTjeneste(
+            tjenestekode = "5516",
+            tjenesteversjon = "4",
+        ),
+        "inkluderingstilskudd" to NAVTjeneste(
+            tjenestekode = "5516",
+            tjenesteversjon = "5",
+        ),
+        "sykefravarstatistikk" to NAVTjeneste(
+            tjenestekode = "3403",
+            tjenesteversjon = gittMiljø.resolve(
+                prod = { "2" },
+                other = { "1" },
+            ),
+        ),
+        "forebyggefravar" to NAVTjeneste(
+            tjenestekode = "5934",
+            tjenesteversjon = "1",
+        ),
+        "rekruttering" to NAVTjeneste(
+            tjenestekode = "5078",
+            tjenesteversjon = "1",
+        ),
+        "tilskuddsbrev" to NAVTjeneste(
+            tjenestekode = "5278",
+            tjenesteversjon = "1",
+        ),
+        "yrkesskade" to NAVTjeneste(
+            tjenestekode = "5902",
+            tjenesteversjon = "1",
+        ),
+    )
+
+    @GetMapping("/api/userInfo/v1")
+    suspend fun getUserInfo(): UserInfoRespons {
+        val organisasjoner = runCatching {
```

Comment:
tenker denne gjerne kan være i en `async` også. Nå skjer vel denne først, så alle de andre i parallell.

Reply:
ellers veldig oversiktlig kode for det parallell!

Reply:
> tenker denne gjerne kan være i en `async` også. Nå skjer vel denne først, så alle de andre i parallell.

ja, det var meningen, men gjorde en glipp :)